### PR TITLE
FC render update

### PR DESCRIPTION
**Pull request by napakalas**
Look for FTU, port, and connector as neuron connectivity terminals.

### CODE REVIEW

File: mapmaker/routing/__init__.py [hunk 1]
```diff
@@ -762,7 +766,7 @@ def __route_graph_from_connectivity(self, path: 'Path', debug=False) -> tuple[nx
                 log.warning(f'{path.id}: {warning}')
 
         # In a case of FC map, we need to remove missing nodes
-        if self.__flatmap.manifest.kind == 'functional':
+        if settings.get('NPO', False):
```
Comment (dbrnz) [inline mapmaker/routing/__init__.py:769]: This still needs to be just for functional maps, i.e:
```
if settings.get('NPO', False) and self.__flatmap.manifest.kind == 'functional':
```

File: tools/npo_alias.py
Comment (dbrnz) [inline tools/npo_alias.py:1]: A merge conflict to resolve...

File: tools/npo_align.py
Comment (dbrnz) [inline tools/npo_align.py:1]: Another merge conflict...

File: tools/npo_check.py
Comment (dbrnz) [inline tools/npo_check.py:1]: Why `str()`? Is it not better to type annotate the function's parameters to be `str` and resolve any type issues when invoking the function?

What is meant by `artefac` (also spelt `artifac` in a help message)??

**Review by dbrnz — COMMENTED** [review]
See comments against files.

File: tools/npo_alias.py
Comment (dbrnz) [inline tools/npo_alias.py:1]: Why are you committing conflicts??

File: mapmaker/routing/__init__.py (hunk 1 above)
Comment (napakalas) [reply mapmaker/routing/__init__.py:769]: Yeah, this should be addressing functional maps only. I'll set this as suggested.

File: tools/npo_alias.py
Comment (napakalas) [reply tools/npo_alias.py:1]: Sorry about this, I unintentionally push my brach before fixing conflicts and your notes. That is why I reset the head to the initial unresolved commit.

Comment (dbrnz) [reply tools/npo_alias.py:1]: That's why it's good practice to have a separate branch for each separate issue, so a PR only has what's relevant to a particular issue and doesn't bring in other bits and pieces... :).

Comment (napakalas) [reply tools/npo_alias.py:1]: Thank you for your advice, now I implement this practice. I will make some PRs but after completing this PR.

File: tools/npo_check.py
Comment (napakalas) [reply tools/npo_check.py:1]: - as suggested, resolving the str issue when invoking the function
- artefac/artifac is a typo of artefact. This is a directory to store the generated map and log. Action: updated to artefact

File: tools/npo_alias.py
Comment (napakalas) [reply tools/npo_alias.py:1]: fixed with a new commit.

**Comment by dbrnz** [comment]
All now good, thanks! I did do a little tidying -- see commit 2676b7d.

**Comment by napakalas** [comment]
Thank you.

**Pull request merged**